src/error_estimation/jump_error_estimator.C

```diff
@@ -183,6 +183,12 @@ void JumpErrorEstimator::estimate_error (const System & system,
       const dof_id_type e_id = e->id();
 
 #ifdef LIBMESH_ENABLE_AMR
+
+#ifdef LIBMESH_ENABLE_INFINITE_ELEMENTS
+      if (e->infinite())
+         continue;
```

Comment:
Should we have a libmesh_warning() here?  Although it's probably good enough in practice, I can't image that this is the *ideal* way to get an error indicator on a mesh with infinite elements, just taking into account jumps between finite elements.

Reply:
yes, probably a warning would be a nice option; on the other hand, if we account for the jump here, it can be caused by insufficient radial base of the infinite element which would, in turn, lead to repeated refinement without getting improvement there.